### ISSUE TITLE
Invalid argument (Errno::EINVAL)

### ISSUE THREAD

**Issue by StudioMaX** [express42/vagrant-foodshow#8]
Quite often after vagrant up or vagrant reload the foodshow chashes with the following error:
```
C:/Users/Pavel/.vagrant.d/gems/gems/vagrant-foodshow-1.1.0/lib/vagrant-foodshow/action/stop.rb:16:in `kill': Invalid argument (Errno::EINVAL)
==> local: ngrok terminated
	from C:/Users/Pavel/.vagrant.d/gems/gems/vagrant-foodshow-1.1.0/lib/vagrant-foodshow/action/stop.rb:16:in `block (2 levels) in call'
	from C:/Users/Pavel/.vagrant.d/gems/gems/vagrant-foodshow-1.1.0/lib/vagrant-foodshow/action/stop.rb:13:in `open'
	from C:/Users/Pavel/.vagrant.d/gems/gems/vagrant-foodshow-1.1.0/lib/vagrant-foodshow/action/stop.rb:13:in `block in call'
	from C:/Users/Pavel/.vagrant.d/gems/gems/vagrant-foodshow-1.1.0/lib/vagrant-foodshow/action/stop.rb:12:in `glob'
	from C:/Users/Pavel/.vagrant.d/gems/gems/vagrant-foodshow-1.1.0/lib/vagrant-foodshow/action/stop.rb:12:in `call'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/warden.rb:34:in `call'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/builder.rb:116:in `call'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/runner.rb:66:in `block in run'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/util/busy.rb:19:in `busy'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/action/runner.rb:66:in `run'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/machine.rb:224:in `action_raw'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/machine.rb:199:in `block in action'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/environment.rb:561:in `lock'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/machine.rb:185:in `call'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/machine.rb:185:in `action'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/plugins/commands/reload/command.rb:39:in `block in execute'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/plugin/v2/command.rb:235:in `block in with_target_vms'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/plugin/v2/command.rb:229:in `each'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/plugin/v2/command.rb:229:in `with_target_vms'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/plugins/commands/reload/command.rb:37:in `execute'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/cli.rb:42:in `execute'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant/environment.rb:302:in `cli'
	from C:/HashiCorp/Vagrant/embedded/gems/gems/vagrant-1.8.1/bin/vagrant:174:in `<main>'
```

I checked manually and can confirm that at this moment ngrok is launched and .pid file exists in directory C:\Users\Pavel\.vagrant.d\tmp\ngrok-3d143ae7-4519-43ed-b7ca-6c9262207d78-8080.pid

In Vagrantfile I use this config
```
  if Vagrant.has_plugin?('vagrant-foodshow')
    config.foodshow.enabled = true
    config.foodshow.authtoken = "***"
    config.vm.network :forwarded_port, guest: 80, host: 8080, ngrok_proto: "http+https"
  end
```

I'm have the following configuration:
ngrok: 2.0.19
vagrant-foodshow: 1.1.0
OS: Windows 10
Vagrant: 1.8.1
VirtualBox: 5.0.14

**Comment by StudioMaX** [comment]
Fixed in https://github.com/express42/vagrant-foodshow/pull/9

**Comment by sample** [comment]
Released 1.1.1 with fix

**Comment by StudioMaX** [comment]
Thanks!
